tilt-setup/deploy_tilt_env.sh

```diff
@@ -76,7 +76,7 @@ spec:
         secret:
           secretName: ${IRONIC_SECRET_NAME}
 EOF
-tools/bin/kustomize build config/tls/
+tools/bin/kustomize build config/components/tls/
```

Comment:
I don't think this is correct. The path points to a kustomize component, which is not meant to be built directly. It should be used by another kustomization. If you try to `kustomize build` this it just gives an empty output.
I think what we should do is either to use the `config/overlays/basic-auth_tls` overlay or create one here on the fly (this is pretty much what the script does already)